### PR TITLE
quality improvement for query_by_election function

### PR DESCRIPTION
**Pull request by N-Perry**

### CODE REVIEW

**Comment by PaDun** [comment]
Looks like some good changes. I particularly appreciate the random comment to recommit.

**Comment by phil0s0pher** [comment]
https://github.com/Unidata/siphon/blob/master/.travis.yml

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ install:
   - pip install -r requirements.txt
 script:
   - pytest
-  - pytest --cov=src tests/
-after_success:
-  - codeclimate-test-reporter
-  
+after_script:
+  - pytest --cov-report xml --cov=src tests/
+  - python-codacy-coverage -r coverage.xml
```
Comment (prestoncarman) [inline .travis.yml]: Add extra new line.

File: README.md [hunk 2]
```diff
@@ -15,8 +15,6 @@ This is a python based REST api that ASWWU web uses for EVERYTHING.
 The following python packages
 (pattern, requests, SQLAlchemy, tornado, bleach, etc.) need to be
 installed and they can be installed with the following command.
-
-Linux:
```
Comment (prestoncarman) [inline README.md]: don't delete this text.

**Pull request merged**